packages/sak-app/src/fagsak/FagsakIndex.tsx

```diff
@@ -134,6 +135,15 @@ const FagsakIndex = () => {
     },
   );
 
+  const { data: direkteOvergangFraInfotrygd } = restApiHooks.useRestApi<OvergangFraInfotrygd>(
+    K9sakApiKeys.DIREKTE_OVERGANG_FRA_INFOTRYGD,
+    {},
+    {
+      updateTriggers: [!behandling],
```

Comment:
trengs denne? behandling vil vel bare være false èn gang?

Reply:
Kan eksperimentere med det, har tatt logikken fra andre steder man gjør api-kall i løsningen. Var noe problem med at det ikke ble kalt eller kalt for tidlig

Reply:
Den trengs, ja. Requesten kjører ikke uten

Reply:
okidoki